package.json

```diff
@@ -32,8 +32,8 @@
     "@polkadot/rpc-provider": "^10.7.3",
     "@polkadot/util": "^12.3.2",
     "@polkadotcloud/core-ui": "0.2.72",
-    "@polkadotcloud/react-odometer": "0.1.17",
-    "@polkadotcloud/utils": "0.2.22",
+    "@polkadotcloud/react-odometer": "file:./packages/@polkadotcloud/react-odometer",
+    "@polkadotcloud/utils": "file:./packages/@polkadotcloud/utils",
```

Comment:
@rosee-xo can you please explain why are we vendoring in a dependency library and which exactly version of it is that ? 

If we need to pin to a specific version of `@polkadotcloud/react-odometer` and `@polkadotcloud/utils` why not just pin the version dependency number in package.json ?